source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,193 @@
+{
+  "retries": {
+    // Configure retries for `cypress run`
+    // Default is 0
+    "runMode": 2,
+    // Configure retries for `cypress open`
+    // Default is 0
+    "openMode": 1
+  }
+}
+```
+
+## Custom Configurations
+
+### Individual Test(s)
+
+If you want a specific test to run a custom number of retries, you can set this by using the {% url "test's configuration" writing-and-organizing-tests#Test-Configuration %}.
+
+```jsx
+// Customize retries for an individual test
+describe('User sign-up and login', () => {
+  // `it` test block with no custom configuration
+  it('should redirect unauthenticated user to sign-in page', () => {
+    // ...
+  })
+
+  // `it` test block with custom configuration
+  it('allows user to login', {
+    retries: {
+      runMode: 2,
+      openMode: 1
+    }
+  }, () => {
+    // ...
+  })
+})
+```
+
+### Test Suite(s)
+
+If you want a suite of tests to re-run a custom number of times, you can do this by setting the suite's configuration.
+
+```jsx
+// Customizing retries for a suite of tests
+describe('User bank accounts', {
+  retries: {
+    runMode: 2,
+    openMode: 1,
+  }
+}, () => {
+  // Individual tests will be run normally
+  it('allows a user to view their transactions, () => {
+    // ...
+  }
+
+  it('allows a user to edit their transactions, () => {
+    // ...
+  }
+})
+```
+
+You can find more information about custom configurations here: {% url "Test Configuration" configuration#Test-Configuration %}
+
+# Screenshots
+
+With test retries, Cypress will take a separate screenshot for each test attempt. These will be properly labeled with the attempt number appended at the end of each filename.
+
+```js
+// Example file names
+'User Sign-up and Login -- should display login errors.png'
+'User Sign-up and Login -- should display login errors (failed).png'
+'User Sign-up and Login -- should display login errors (attempt 2).png'
+'User Sign-up and Login -- should display login errors (failed) (attempt 2).png'
+'User Sign-up and Login -- should display login errors (attempt 3).png'
+'User Sign-up and Login -- should display login errors (failed) (attempt 3).png'
+```
+
+# Dashboard
+
+If you are using the {% url "Cypress Dashboard" dashboard %}, test retries will not change your current experience at this time. However, if you are interested in future enhancements coming to the Dashboard experience, be sure to check out our {% url "product roadmap" https://cypress-io.productboard.com/roadmap/1238172-product-roadmap %}.
+
+# Tips and Strategies
+
+While test retries are great for helping to avoid false negatives from failing an entire test run, it is not a good replacement for writing good tests. Here are some tips and strategies to keep in mind in order to maximize the effectiveness of your tests with test retries:
+
+- If you are noticing that you need to increase the number of retries, the cause is more likely due to how the tests are written and is worth spending the time to investigate.
```

Comment:
This section is very weak, almost so much so that I would recommend removing it. If we want to offer strategies this should be stronger - right now it's like 'write better tests' 🤷 .

Reply:
agreed ^

Reply:
The section is trying to explain why you might adopt test retries in your test suite and to beware of the bandaid solution that retries _could_ be. I think this message is important to explain to users, but perhaps isn't needed right now.